test/docfx.Test/lsp/LanguageServerTestCommand.cs

```diff
@@ -16,6 +16,8 @@ public class LanguageServerTestCommand
 
         public Dictionary<string, JToken> ExpectDiagnostics { get; init; }
 
+        public GetCredentialRequest ExpectGetCredentialRequest { get; init; }
```

Comment:
Move `Response` outside of `GetCredentialRequest` to simplify the structure, the `Response` field could also potentially be shared with other `expectXYZRequest` calls

```suggestion
        public JToken ExpectGetCredentialRequest { get; init; }
        public JToken Response { get; init; }
```